static_compress/mixin.py

```diff
@@ -34,7 +35,7 @@ def __init__(self, *args, **kwargs):
         self.allowed_extensions = getattr(settings, "STATIC_COMPRESS_FILE_EXTS", ["js", "css", "svg"])
         self.compress_methods = getattr(settings, "STATIC_COMPRESS_METHODS", DEFAULT_METHODS)
         self.keep_original = getattr(settings, "STATIC_COMPRESS_KEEP_ORIGINAL", True)
-        self.minimum_kb = getattr(settings, "STATIC_COMPRESS_MIN_SIZE_KB", 30)
+        self.minimum_kb = getattr(settings, "STATIC_COMPRESS_MIN_SIZE_KB", 0)
```

Comment:
I'm not ok with changing this. Compressing everything make building slower and it is not always smaller.

If you want deterministic file hash from your web server, either fix the web server or change the library's configuration value.

Reply:
You can either turns gzip off in your web server, or set STATIC_COMPRESS_MIN_SIZE_KB=0 in your site's configuration.

Reply:
yes, its right

Reply:
but with >0 not to exclude that it comes to the 404 error with default settings. so consider the user your module faulty and uninstall it:)

Reply:
Are you hardcoding .gz/.br in your page?

The [installation guide](https://github.com/whs/django-static-compress#installation) already recommends that your web server should be configured to serve precompressed files.

Reply:
Nope. I am using `CompressedStaticFilesStorage` und yes `gzip_static` is `on`.

Reply:
If `gzip` is `off`, i get 404 because requested file is `C.hashC.css` but exists as gzipped only `A.OriginalHash.css`
A, B are intermediate und C final versions of a certain file.